src/ResourceManager/Storage/Commands.Management.Storage/help/Get-AzureRmStorageAccountNetworkRuleSet.md

```diff
@@ -18,21 +18,21 @@ Get-AzureRmStorageAccountNetworkRuleSet [-ResourceGroupName] <String> [-Name] <S
 ```
 
 ## DESCRIPTION
-The **Get-AzureRmStorageAccountNetworkRuleSet** cmdlet gets the NetworkRule property of a Storage Account
+The **Get-AzureRmStorageAccountNetworkRuleSet** cmdlet gets the NetworkRule property of a Storage account
 
 ## EXAMPLES
 
-### Example 1: Get NetworkRule property of a specified storage account
+### Example 1: Get NetworkRule property of a specified Storage account
 ```
 PS C:\> Get-AzureRmStorageAccountNetworkRuleSet  -ResourceGroupName "rg1" -AccountName "mystorageaccount"
 ```
 
-This command gets NetworkRule property of a specified storage account
+This command gets NetworkRule property of a specified Storage account
 
 ## PARAMETERS
 
 ### -DefaultProfile
-The credentials, account, tenant, and subscription used for communication with azure.
```

Comment:
Something happened here - DefaultProfile should not be changed here.  Can you regenerate this file?